README.md

```diff
@@ -220,12 +216,17 @@ To understand what we're trying to become, it may be useful to understand what
 we're trying to avoid. See effects of [contempt](http://blog.aurynn.com/86/contempt-culture)
 and [competition](https://medium.com/tech-diversity-files/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#.yjrp4mqkb) on environments.
 
-#### Favour experimentation over argument
+#### Favour experimentation over opinion
```

Comment:
My feeling is that argument and experimentation contrast better - we don't want people to argue, we want them to experiment. Experiments come with opinions, and having opinions is good :)